ddpg:
  alpha: 0.00005
  beta: 0.0005
  gamma: 0.99
  tau: 0.001
  batch_size: 64
  max_size: 10000
  n_episodes: 1000
  n_steps: 500
  layer1_size: 400
  layer2_size: 300
  plot_file_name: 'AirSim.png'